homework.txt

```diff
@@ -0,0 +1,27 @@
+//I couldn't get my code to function.
+function opt(num1,num2,op){
+	
+var add, subtract, multiply, divide = "";
+var num1, num2 = "";
+
+	
+	if (op === add) {
+	return (num1+num2)
+        }
+
+	if (op === subtract){
+		console.log(num1 - num2);
+        }
+
+	if (op === multiply){
+		console.log(num1 * num2);
+        }
+
+	if (op === divide){
+		console.log(num1 / num2)
+        }
+
+	else {
+		console.log("ERROR")
+        }
+}
```

Comment:
Almost got it there @abduwib , the reason this wasn't functioning is that `add`, `subtract`, .. didn't hold any value in them. So comparing `op === add` will just check if `op` is empty. What you're looking for is something similar to
```js
function opt(num1,num2,op){

	if (op === "add") {
	return (num1+num2)
        }

	if (op === "subtract"){
		console.log(num1 - num2);
        }

	if (op === "multiply"){
		console.log(num1 * num2);
        }

	if (op === "divide"){
		console.log(num1 / num2)
        }

	else {
		console.log("ERROR")
        }
}
```